docs/stack_config.rst

```diff
@@ -39,6 +39,44 @@ A list of arbitrary shell or python commands or scripts to run. Find out more in
 
 A dictionary of key-value pairs to be supplied to a CloudFormation or Troposphere template as parameters. The keys must match up with the name of the parameter, and the value must be of the type as defined in the template. Note that Boto3 throws an exception if parameters are supplied to a template that are not required by that template. Resolvers can be used to add functionality to this key. Find out more in the `Resolvers`_ section.
 
+Parameters can be specified as a single or list of values/resolvers. Lists of values/resolvers will be formatted into an AWS compatible comma separated string e.g. "value1,value2,value3". Lists can contain a mixture of values and resolvers.
```

Comment:
Could we make it:

Parameters can be specified as a single **value/resolver** or **a** list of values/resolvers.

It's slightly ambiguous as is: 

Parameters can be specified as a single (?) or parameters can be supplied as a list of values/resolvers